Stratis.Bitcoin.IntegrationTests/RPCTEsts.cs

```diff
@@ -118,5 +123,98 @@ public void CanGetPeersInfo()
                 }
             }
         }
+
+        /// <summary>
+        /// Can call getpeersinfo by string arguments
+        /// </summary>
+        [Fact]
+        public void CanGetPeersInfoByStringArgs()
+        {
+            using (NodeBuilder builder = NodeBuilder.Create())
+            {
+                CoreNode nodeA = builder.CreateStratisPowNode();
+                builder.StartAll();
+                RPCClient rpc = nodeA.CreateRPCClient();
+                using (Node nodeB = nodeA.CreateNodeClient())
+                {
+                    nodeB.VersionHandshake();
+                    var resp = rpc.SendCommand("getpeerinfo").ResultString;
+                    Assert.True(resp.StartsWith("[\r\n  {\r\n    \"id\": 0,\r\n    \"addr\": \"["));
```

Comment:
this looks like we are very much depending on having exact indentation on the output
the result should be JSON right? valid JSON response can have arbitrary indentation
i don't think this should be tested like that.

Reply:
For this to be working in Linux, Windows and OSX, it's probably s good idea to use Environment.NewLine instead of \r\n.